src/Index.cmake

```diff
@@ -1,7 +1,3 @@
-cmake_minimum_required(VERSION 3.20)
```

Comment:
this is the real min version! Do not change.

Reply:
project_options doesn't work with older versions. Please revret